docker.d/scriptworker.yml

```diff
@@ -28,5 +28,4 @@ task_log_dir: { "$eval": "TASK_LOGS_DIR" }
 watch_log_file: true
 scriptworker_provisioners:
     - { "$eval": "PROVISIONER_ID" }
-    - scriptworker-prov-v1
```

Comment:
This is cleanup, we no longer need this.

Reply:
This change broke the mac signing pool.

Reply:
@escapewindow Bah, forgot about mac signing pool, sorry about that :(